usaspending_api/api_contracts/contracts/award/idv/Download.md

```diff
@@ -0,0 +1,34 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# IDV Download [/api/v2/download/idvs/]
+
+This end point returns a zipped file containing IDV data.
+
+## POST
+
++ Request (application/json)
+    + Parameters
+        + `award_id`: `CONT_AW_9700_-NONE-_N0018918D0057_-NONE-` (required, string)
++ Response 200 (application/json)
+    + Attributes
+        + results (IDVDownloadResponse)
+
+# Data Structures
+
+## IDVDownloadResponse (object)
++ total_size: 35.055 (nullable, number)
+    The total size of the file being returned
++ file_name: `012_account_balances_20180613140845.zip` (required, string)
++ total_rows: 652 (nullable, number)
++ total_columns: 27 (nullable, number)
++ url: `xyz/path_to/bucket/012_account_balances_20180613140845.zip` (required, string)
+    Where the file lives in S3
++ message (optional, nullable)
++ status: `finished` (required, enum[string])
```

Comment:
Dredd doesn't like `finished` here.  Change to `+ status (required, enum[string])` and we should be good to go.

Reply:
Was reusing pattern in `DownloadCenter.md` will see if the mocks work when this doesn't have a value.

![image](https://user-images.githubusercontent.com/12897813/56426904-eabef180-6287-11e9-9a18-7780d2174719.png)

Reply:
That's fine.  I believe most of the older contracts have not gone through Dredd validation so there will be some back and forth as we try to bring everything into compliance.

👍

Reply:
It worked as you requested it when executing the mock server with `drakov` so I responded to this change request.